App crashes on open, latest CI, App Build #1547

### Describe the bug

1. Install latest debug CI build, App Build #1547.
2. Open app
3. App crashes

### Logs

```shell
logcat crash:
    E/ACRA    (19179): ACRA caught a RuntimeException for org.jellyfin.androidtv.debug
    E/ACRA    (19179): java.lang.RuntimeException: Unable to start activity ComponentInfo{org.jellyfin.androidtv.debug/org.jellyfin.androidtv.ui.startup.StartupActivity}: org.jellyfin.sdk.api.client.exception.MissingBaseUrlException: Required value baseUrl is null. Provide it by setting ApiClient.baseUrl.
    E/ACRA    (19179): 	at android.app.ActivityThread.performLaunchActivity(ActivityThread.java:2480)
    E/ACRA    (19179): 	at android.app.ActivityThread.handleLaunchActivity(ActivityThread.java:2548)
    E/ACRA    (19179): 	at android.app.ActivityThread.access$800(ActivityThread.java:160)
    E/ACRA    (19179): 	at android.app.ActivityThread$H.handleMessage(ActivityThread.java:1321)
    E/ACRA    (19179): 	at android.os.Handler.dispatchMessage(Handler.java:102)
    E/ACRA    (19179): 	at android.os.Looper.loop(Looper.java:135)
    E/ACRA    (19179): 	at android.app.ActivityThread.main(ActivityThread.java:5604)
    E/ACRA    (19179): 	at java.lang.reflect.Method.invoke(Native Method)
    E/ACRA    (19179): 	at java.lang.reflect.Method.invoke(Method.java:372)
    E/ACRA    (19179): 	at com.android.internal.os.ZygoteInit$MethodAndArgsCaller.run(ZygoteInit.java:984)
    E/ACRA    (19179): 	at com.android.internal.os.ZygoteInit.main(ZygoteInit.java:779)
    E/ACRA    (19179): Caused by: org.jellyfin.sdk.api.client.exception.MissingBaseUrlException: Required value baseUrl is null. Provide it by setting ApiClient.baseUrl.
    E/ACRA    (19179): 	at org.jellyfin.sdk.api.client.ApiClient.createUrl(ApiClient.kt:66)
    E/ACRA    (19179): 	at org.jellyfin.sdk.api.client.ApiClient.createUrl$default(ApiClient.kt:60)
    E/ACRA    (19179): 	at org.jellyfin.sdk.api.client.KtorClient.request$suspendImpl(KtorClient.kt:62)
    E/ACRA    (19179): 	at org.jellyfin.sdk.api.client.KtorClient.request(KtorClient.kt)
    E/ACRA    (19179): 	at org.jellyfin.sdk.api.operations.UserApi.getCurrentUser(UserApi.kt:449)
    E/ACRA    (19179): 	at org.jellyfin.androidtv.auth.SessionRepositoryImpl$setCurrentSession$1.invokeSuspend(SessionRepository.kt:129)
    E/ACRA    (19179): 	at kotlin.coroutines.jvm.internal.BaseContinuationImpl.resumeWith(ContinuationImpl.kt:33)
    E/ACRA    (19179): 	at kotlinx.coroutines.DispatchedTask.run(DispatchedTask.kt:106)
    E/ACRA    (19179): 	at kotlinx.coroutines.EventLoopImplBase.processNextEvent(EventLoop.common.kt:279)
    E/ACRA    (19179): 	at kotlinx.coroutines.BlockingCoroutine.joinBlocking(Builders.kt:85)
    E/ACRA    (19179): 	at kotlinx.coroutines.BuildersKt__BuildersKt.runBlocking(Builders.kt:59)
    E/ACRA    (19179): 	at kotlinx.coroutines.BuildersKt.runBlocking(Unknown Source)
    E/ACRA    (19179): 	at kotlinx.coroutines.BuildersKt__BuildersKt.runBlocking$default(Builders.kt:38)
    E/ACRA    (19179): 	at kotlinx.coroutines.BuildersKt.runBlocking$default(Unknown Source)
    E/ACRA    (19179): 	at org.jellyfin.androidtv.auth.SessionRepositoryImpl.setCurrentSession(SessionRepository.kt:127)
    E/ACRA    (19179): 	at org.jellyfin.androidtv.auth.SessionRepositoryImpl.setCurrentSession$default(SessionRepository.kt:113)
    E/ACRA    (19179): 	at org.jellyfin.androidtv.auth.SessionRepositoryImpl.restoreDefaultSession(SessionRepository.kt:65)
    E/ACRA    (19179): 	at org.jellyfin.androidtv.JellyfinApplication$onCreate$1.onStart(JellyfinApplication.kt:53)
    E/ACRA    (19179): 	at androidx.lifecycle.FullLifecycleObserverAdapter.onStateChanged(FullLifecycleObserverAdapter.java:39)
    E/ACRA    (19179): 	at androidx.lifecycle.LifecycleRegistry$ObserverWithState.dispatchEvent(LifecycleRegistry.java:354)
    E/ACRA    (19179): 	at androidx.lifecycle.LifecycleRegistry.forwardPass(LifecycleRegistry.java:265)
    E/ACRA    (19179): 	at androidx.lifecycle.LifecycleRegistry.sync(LifecycleRegistry.java:307)
    E/ACRA    (19179): 	at androidx.lifecycle.LifecycleRegistry.moveToState(LifecycleRegistry.java:148)
    E/ACRA    (19179): 	at androidx.lifecycle.LifecycleRegistry.handleLifecycleEvent(LifecycleRegistry.java:134)
    E/ACRA    (19179): 	at androidx.lifecycle.ProcessLifecycleOwner.activityStarted(ProcessLifecycleOwner.java:112)
    E/ACRA    (19179): 	at androidx.lifecycle.ProcessLifecycleOwner$2.onStart(ProcessLifecycleOwner.java:83)
    E/ACRA    (19179): 	at androidx.lifecycle.ReportFragment.dispatchStart(ReportFragment.java:88)
    E/ACRA    (19179): 	at androidx.lifecycle.ReportFragment.onStart(ReportFragment.java:108)
    E/ACRA    (19179): 	at android.app.Fragment.performStart(Fragment.java:2077)
    E/ACRA    (19179): 	at android.app.FragmentManagerImpl.moveToState(FragmentManager.java:922)
    E/ACRA    (19179): 	at android.app.FragmentManagerImpl.moveToState(FragmentManager.java:1067)
    E/ACRA    (19179): 	at android.app.FragmentManagerImpl.moveToState(FragmentManager.java:1049)
    E/ACRA    (19179): 	at android.app.FragmentManagerImpl.dispatchStart(FragmentManager.java:1874)
    E/ACRA    (19179): 	at android.app.Activity.performStart(Activity.java:6048)
    E/ACRA    (19179): 	at android.app.ActivityThread.performLaunchActivity(ActivityThread.java:2443)
    E/ACRA    (19179): 	... 10 more
```


### Application version

App Build #1547

### Where did you install the app from?

Sideloaded APK

### Device information

Firestick v1 AFTM

### Android version

Fire OS 5

### Jellyfin server version

10.7.7

--- a/app/src/main/java/org/jellyfin/androidtv/auth/SessionRepository.kt
+++ b/app/src/main/java/org/jellyfin/androidtv/auth/SessionRepository.kt
@@ -126,8 +126,12 @@ class SessionRepositoryImpl(
 
 		if (success) runBlocking {
 			userApiClient.applySession(session, deviceInfo)
-			val user by userApiClient.userApi.getCurrentUser()
-			userRepository.updateCurrentUser(user)
+			if (session != null) {
+				val user by userApiClient.userApi.getCurrentUser()
+				userRepository.updateCurrentUser(user)
+			} else {
+				userRepository.updateCurrentUser(null)
+			}
 			preferencesRepository.onSessionChanged()
 			_currentSession.value = session
 		} else {

